### ISSUE TITLE
About text features.

### ISSUE THREAD

**Issue by EdenGabriel** [IMCCretrieval/MomentDiff#3]
Hello, thanks for your interesting work. Can you please provide the Glove text features?

**Comment by EdenGabriel** [comment]
btw, when i utilize the script about training qvhighlights as follows, i get the best result about `"MR-full-mAP": 30.74`. There have a large margin compare with your paper, maybe can u help me adjust the settings about the QVHighlights. Thank you very much. 
```shell

dset_name=hl
ctx_mode=video_tef
train_path=data/highlight/highlight_train_release.jsonl
eval_path=data/highlight/highlight_val_release.jsonl
eval_split_name=val
v_feat_dim=2816
t_feat_dir=/ssd2/mr_hd/qvhighlights/clip_text_features
t_feat_dim=512
results_root=results
exp_id=exp
bsz=32


PYTHONPATH=$PYTHONPATH:. python momentdiff/train.py \
--dset_name ${dset_name} \
--ctx_mode ${ctx_mode} \
--train_path ${train_path} \
--eval_path ${eval_path} \
--eval_split_name ${eval_split_name} \
--v_feat_dirs  /ssd2/mr_hd/qvhighlights/clip_features /ssd2/mr_hd/qvhighlights/slowfast_features \
--v_feat_dim ${v_feat_dim} \
--t_feat_dir ${t_feat_dir} \
--t_feat_dim ${t_feat_dim} \
--bsz ${bsz} \
--results_root ${results_root} \
--exp_id ${exp_id} \
--eval_split_name val \
--seed 42 \
--lr 0.0002 \
--num_workers 12 \
--lr_drop 40 \
--clip_length 2 \
--lw_saliency 4.0 \
--giou_loss_coef 1.0 \
--max_v_l -1
```

**Comment by lpdone** [comment]
Thanks for your attention. I think you can try setting the  span number to 10 and the learning rate to 0.0001.  

We are very sorry that text features cannot be released recently due to data approval reasons in the company. You can try NLTKTokenizer to extract Glove features. You can also refer to https://github.com/minghangz/cpl/tree/main/data.

**Comment by EdenGabriel** [comment]
Thank you for your replay. I'll try.

**Comment by YunjinYeung** [comment]
Hi, when I completed the process of reproduce QVHighlight according to the help of the issue, I have set the loss_bce parameter to 0.1 and lr_rate to 0.0001, and set both span_loss_coef and set_cost_span to 10. But unfortunately, the best result of "MR-full-R1@0.5" is 50.84, which is far from the result of your paper. This is my opt.json
[opt.json.zip](https://github.com/IMCCretrieval/MomentDiff/files/14414622/opt.json.zip)
file, may I ask if there are any important parameters I have not adjusted?
